rosidl_runtime_rs/src/sequence.rs

```diff
@@ -445,7 +466,25 @@ where
 }
 
 // ========================= impl for SequenceIterator =========================
-
+/// Implements the `Clone` trait for the `Sequence` struct.
```

Comment:
This comment doesn't match the code, it implements the `Iterator` trait (see https://doc.rust-lang.org/std/iter/trait.Iterator.html#tymethod.next)

Reply:
This still does not match the code, it says "Implements the `Clone` trait for the `Sequence` struct.", but the traits that's bein implemented is `Iterator` (see https://github.com/ros2-rust/ros2_rust/pull/409/files#diff-4577ff1d223537bc123ec0d170a0a48ac7621b9a5e0e4fa1fae114da05edff4dR467)